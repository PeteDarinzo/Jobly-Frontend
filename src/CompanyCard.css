
.CompanyCard {
  text-align: left;
  margin: 0 auto 25px;
  text-decoration: none;
  color: whitesmoke;
  background-color: darkslategray;
  width: 55%;
  padding: 5px;
  padding-left: 50px;
}