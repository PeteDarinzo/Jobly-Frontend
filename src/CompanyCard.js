import React from "react";
import "./Card.css";


const CompanyCard = ({ name, description }) => {

  return (
    <div className="Card shadow p-3">
      <h2>{name}</h2>
      <p>{description}</p>
    </div>
  );

}

export default CompanyCard;