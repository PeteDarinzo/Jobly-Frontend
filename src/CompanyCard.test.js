import CompanyCard from "./CompanyCard";
import { render, fireEvent } from '@testing-library/react';
import { MemoryRouter } from "react-router-dom"

it("renders home with crashing", () => {
  render(
    <MemoryRouter>
      <CompanyCard name="test company" description="a test company" />
    </MemoryRouter>
  );
});

it("matches the home snapshot", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <CompanyCard name="test company" description="a test company" />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});