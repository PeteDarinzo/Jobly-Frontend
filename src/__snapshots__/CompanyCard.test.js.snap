// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the home snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card shadow p-3"
  >
    <h2>
      test company
    </h2>
    <p>
      a test company
    </p>
  </div>
</DocumentFragment>
`;
